lib/modules/manager/sbt/extract.ts

```diff
@@ -302,7 +421,9 @@ function parseSbtLine(
       isMultiDeps = true;
     } else if (isMultiDeps) {
       const rightPart = line.replace(regEx(/^[\s,]*/), '');
-      const depExpr = rightPart.replace(regEx(/[\s,]*$/), '');
+      const depExpr = rightPart
+        .replace(regEx(/[\s,]*$/), '')
+        .replace(/\s*(force|withSources|exclude(All)?).*$/, '');
```

Comment:
```suggestion
        .replace(regEx(/\s*(?:force|withSources|exclude(?:All)?).*$/, ''));
```
- use our regex wrapper 
- prefer non-capturing groups